.travis.yml

```diff
@@ -1,24 +1,18 @@
 sudo: false
 language: node_js
+node_js:
+  - '5'
 cache:
   directories:
     - node_modules
-notifications:
-  email: false
-node_js:
-  - '4'
-before_install:
-  - npm i -g npm@^3.0.0
 before_script:
   - npm prune
+notifications:
+  email: false
 script:
   - npm run validate
 after_success:
-  - 'curl -Lo travis_after_all.py https://git.io/travis_after_all'
-  - python travis_after_all.py
-  - 'export $(cat .to_export_back) &> /dev/null'
```

Comment:
These are not required, since there are no multiple matrix https://github.com/dmakhno/travis_after_all.
Please correct me, if I have misunderstood.